rules/prefer-modern-dom-apis.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
```

Comment:
rebase this PR, 
run `yarn lint rules/prefer-modern-dom-apis.js --fix`
you'll get an error here, and will be fixed

Reply:
sorry this will not fix currectly, after rebase, rename it to `getDocumentationUrl`

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/d638e5477d7240ea777dcaf249d0121a5a1910eb/rules/catch-error-name.js#L4